### PR TITLE
Add Gentoo

### PR DESCRIPTION
**Pull request by TheEvilSkeleton**
Closes https://github.com/mirukana/mirage/issues/135

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -83,6 +84,20 @@ in this example [yay](https://github.com/Jguer/yay) for the release version:
 
     yay -S matrix-mirage
 
+#### Gentoo
+
+Available in the [src_prepare overlay](https://gitlab.com/src_prepare/src_prepare-overlay)
+- [releases](https://gitlab.com/src_prepare/src_prepare-overlay/-/tree/master/net-im/mirage)
+- [git master](https://gitlab.com/src_prepare/src_prepare-overlay/-/blob/master/net-im/mirage/mirage-9999.ebuild)
+
+Installing Mirage
```
Comment (mirukana) [inline INSTALL.md:93]:
```suggestion
Installing Mirage:
```

File: INSTALL.md [hunk 2]
```diff
@@ -83,6 +84,20 @@ in this example [yay](https://github.com/Jguer/yay) for the release version:
...
+
+1. [Add the overlay](https://gitlab.com/src_prepare/src_prepare-overlay#adding-the-overlay)
+2. [Unmask](https://wiki.gentoo.org/wiki/Knowledge_Base:Unmasking_a_package) `net-im/mirage`
+3. Run:
+
+        emerge net-im/mirage
+
```
Comment (mirukana) [inline INSTALL.md:100]:
```suggestion
3. Run `emerge net-im/mirage`
```

**Review by mirukana — CHANGES_REQUESTED** [review]
Just some formatting nitpicks:

**Comment by TheEvilSkeleton** [comment]
Looks good to me

**Comment by TheEvilSkeleton** [comment]
> Just some formatting nitpicks:

It's okay, that's something we have in common :)

**Pull request merged**